src/wasm-type.h

```diff
@@ -314,10 +315,10 @@ class Type {
 
   // Construct from a heap type description. Also covers construction from
   // Signature, Struct or Array via implicit conversion to HeapType.
-  Type(HeapType heapType, Nullability nullable)
-    : Type(heapType.getID() | (nullable == Nullable ? NullMask : 0)) {
-    assert(heapType.isBasic() ||
```

Comment:
why remove the `isBasic` part?

Reply:
It's redundant since basic heap types should always satisfy the right hand side, anyway.